tests/playwright/src/spec/minikube-extension.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { 
+    expect as playExpect, 
+    ExtensionsPage,  
+    test, 
+    waitForPodmanMachineStartup, 
+} from '@podman-desktop/tests-playwright';
+
+const EXTENSION_IMAGE: string = 'ghcr.io/podman-desktop/podman-desktop-extension-minikube:nightly';
+const EXTENSION_NAME: string = 'minikube';
+const EXTENSION_LABEL: string = 'podman-desktop.minikube';
+
+let extensionsPage: ExtensionsPage; 
+
+const skipExtensionInstallation = process.env.SKIP_EXTENSION_INSTALL ? process.env.SKIP_EXTENSION_INSTALL : false;
```

Comment:
`= process.env.SKIP_EXTENSION_INSTALL === 'true'` should be sufficient and correct.